packages/audit/package.json

```
@@ -13,8 +13,7 @@
   },
   "scripts": {
     "lint": "eslint -c ../../eslint.json src/**/*.ts test/**/*.ts",
-    "_test": "cd ../.. && c8 --reporter lcov --reports-dir packages/audit/coverage ts-node packages/audit/test --type-check",
-    "test": "pnpm run compile && pnpm run _test",
+    "test": "jest",
```

maybe try:

```suggestion
    "_test": "jest",
    "test": "pnpm run compile && pnpm run _test",
```

also, is coverage calculated by default? we need coverage